src/Evolution/DiscontinuousGalerkin/DgElementArray.hpp

```diff
@@ -45,7 +50,12 @@ CREATE_HAS_STATIC_MEMBER_VARIABLE_V(use_z_order_distribution)
  * in the `Metavariables`, in which case elements are assigned to processors via
  * round-robin assignment. In both cases, an unordered set of `size_t`s can be
  * passed to the `allocate_array` function which represents physical processors
- * to avoid placing elements on.
+ * to avoid placing elements on. If the space-filling curve is used, then if
+ * `static constexpr bool local_time_stepping = true;` is specified
```

Comment:
isn't it the reverse? I think this should be `false`.

Also, why prefer `Uniform` over `NumGridPoints` for global time stepping?

Reply:
yes, good catch